src/logion/controllers/collection.controller.ts

```diff
@@ -261,4 +265,68 @@ export class CollectionController extends ApiController {
         const authenticated = await this.authenticationService.authenticatedUser(this.request);
         await this.checkCanDownload(authenticated, collectionLocId, itemId, hash);
     }
+
+    static getLatestDeliveries(spec: OpenAPIV3.Document) {
+        const operationObject = spec.paths["/api/collection/{collectionLocId}/{itemId}/latest-deliveries"].get!;
+        operationObject.summary = "Provides information about the latest copies delivered to the item's token owner";
+        operationObject.description = "This is a public resource";
+        operationObject.responses = getDefaultResponsesWithAnyBody();
+        setPathParameters(operationObject, {
+            'collectionLocId': "The ID of the Collection LOC",
+            'itemId': "The ID of the Collection Item",
+        });
+    }
+
+    @HttpGet('/:collectionLocId/:itemId/latest-deliveries')
+    @Async()
+    async getLatestDeliveries(_body: any, collectionLocId: string, itemId: string, hash: string): Promise<ItemDeliveriesResponse> {
```

Comment:
Param `hash` is not present in the path.

Reply:
Indeed, thanks!